app/webpacker/components/RegistrationsV2/Register/StripeWrapper.jsx

```diff
@@ -45,6 +46,15 @@ export default function StripeWrapper({
   return (
     <>
       <Header>Payment</Header>
+      <Message positive>
+        {I18n.t('registrations.payment_form.hints.payment_button')}
+      </Message>
+      { data?.api_amounts.stripe > 2 * initialAmount
```

Comment:
You shouldn't rely on Stripe-specific values here. I think since we have the `initialAmount` from the backend already (which is guaranteed to be ISO because it's served through our backend money gem) you can just use `donationAmount > initialAmount` as condition. What do you think?

Reply:
sure